config/cross_section_config.py

```diff
@@ -552,7 +552,7 @@ def __fill_defaults_13TeV__( self ):
         path_to_files = self.path_to_files
 
         # self.new_luminosity = 40.028  # pb^-1
-        self.new_luminosity = 41.62912  # pb^-1
+        self.new_luminosity = 71.515  # pb^-1
```

Comment:
Is this luminosity after the 4% correction from PPD?  Or do who not have to do that anymore?  For the first 42\pb, brilcalc (or lcr2.py) would return ~40\pb, but the actual lumi was 40 * 1.04 = 42\pb